src/BusyIndicator/BusyIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { BUSY_INDICATOR_SIZES } from '../utils/constants';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+const BusyIndicator = React.forwardRef(({
+    children,
+    className,
+    show,
+    size,
+    disableStyles,
+    ...props
+}, ref) => {
+
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/icon.css');
+            require('fundamental-styles/dist/busy-indicator.css');
+        }
+    }, []);
+
+    const busyIndicatorClasses = classnames(
+        {
+            [`fd-busy-indicator--${size}`]: !!size
+        },
+        className
+    );
+
+    if (!show)
+        return (<></>);
+
+    return (
+        <div
+            {...props}
+            className={busyIndicatorClasses}
+            ref={ref} >
```

Comment:
[a11y]
Should we default an aria-label attribute?  Something like
```js
 aria-label={localizedText.loading}
```
and should `aria-hidden="false"` be applied by default?

Reply:
@meganaconley ^

Reply:
Added above suggestions.
Did not add `aria-hidden` in propDescription as its implicit.